test/helpers/create2.js

```diff
@@ -0,0 +1,29 @@
+
+// deterministically computes the smart contract address given
+// the account the will deploy the contract (factory contract)
+// the salt as uint256 and the contract bytecode
+const buildCreate2Address = function (creatorAddress, saltHex, byteCode) {
+  return web3.utils.toChecksumAddress(`0x${web3.utils.sha3(`0x${[
+    'ff',
+    creatorAddress,
+    saltHex,
+    web3.utils.soliditySha3(byteCode),
+  ].map(x => x.replace(/0x/, '')).join('')}`).slice(-40)}`);
+};
+
+// encodes parameter to pass as contract argument
+const encodeParam = function (dataType, data) {
```

Comment:
I'd remove `encodeParam` and `isContract` if we won't be needing them for the tests.

General-use test helpers should be added to [openzeppelin-test-helpers](https://github.com/OpenZeppelin/openzeppelin-test-helpers).